dotnet/src/dotnetframework/GxClasses/Domain/GxHttpClient.cs

```diff
@@ -819,9 +818,9 @@ public void HttpClientExecute(string method, string name)
 					_errDescription = aex.InnerException.Message;
 				else
 					_errDescription = aex.Message;
-				response = new HttpResponseMessage();
-				response.Content = new StringContent(HttpHelper.StatusCodeToTitle(HttpStatusCode.InternalServerError));
-				response.StatusCode = HttpStatusCode.InternalServerError;
+				_chunkedResponse = new HttpResponseMessage();
```

Comment:
Why response was renamed to "chunkedResponse". Shouldn't we keep the original name "response" ? Maybe I'm missing something

Reply:
You're correct about the naming. The 'chunked' prefix was indeed unnecessary. The former variable 'response' was a local routine variable, whereas the new 'response' variable is now an instance variable. I've simply added the '_' prefix to the previous name 'response' to distinguish it from local variables.